roles/chrony/tasks/validate.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Validate file for chrony role
+
+- name: Validate if the public_ntp_servers is not none
+  tags: [chrony, chrony-validate]
+  assert:
+    that: public_ntp_servers is not none
+    fail_msg: "public_ntp_servers is not set"
+    success_msg: "The item public_ntp_servers is defined"
+
+- name: Load the Debian specific defaults
```

Comment:
I'd move these includes in `main.yml`. So it will be easier to follow the code flow.